### PR TITLE
#v2 documentation

### PR DESCRIPTION
**Pull request by ColeFeely6**
Fix Search Engine to have no ads and use Microshare colors. Also includes a hard code to fix the previous issues rather than through google. I also worked on a Google analytics script to view most searched for pages and observe if there are any incorrect responses from the engine. I do believe that some return pages are incorrect, would like to find them.

Additionally, adding a bunch more pages for the dashboard guide update. The dashboard guide update will not be in this pull, the focus of the pull request is to fix that search engine!

### CODE REVIEW

File: _layouts/docs.html [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
   <head>
-    {% include header.html %}
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+      <script async src="https://www.googletagmanager.com/gtag/js?id=UA-173448502-1"></script>
+      <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+
+        gtag('config', 'UA-173448502-1');
```
Comment (tpanagos) [inline _layouts/docs.html:11]: What google account is this tied to?

File: index.html [hunk 2]
```diff
@@ -53,34 +64,35 @@ <h1 class="display-4">Documentation</h1>
 
 </head>
 <body>
-    
-  <div class="container justify-content-md-center">
-    <script async src="https://cse.google.com/cse.js?cx=010028121289637253157:oyhvms8bdqm"></script>
+    <div class="container justify-content-md-center">
+    <script async src="https://cse.google.com/cse.js?cx=001595182637908644242:njcczvsmzd4"></script>
```
Comment (tpanagos) [inline index.html:68]: Where does these keys come from?

File: _layouts/docs.html (hunk 1 above)
Comment (ColeFeely6) [reply _layouts/docs.html:11]: My personal one. I was thinking about setting up an account just for Microshare so anyone can use it. I can easily swap it out.

File: index.html (hunk 2 above)
Comment (ColeFeely6) [reply index.html:68]: The keys come from the Google Programmable Search Engine [https://programmablesearchengine.google.com/](https://programmablesearchengine.google.com/).

Comment (tpanagos) [reply index.html:68]: Use this one:
<script async src="https://cse.google.com/cse.js?cx=005501767810126761962:bnqqnli8vek"></script>
<div class="gcse-search"></div>

**Pull request merged**